libraries/lib-module-manager/PluginManager.cpp

```diff
@@ -577,6 +441,29 @@ PluginManager::~PluginManager()
    Terminate();
 }
 
+void PluginManager::InitializePlugins()
```

Comment:
Too much is happening in the commit "PluginValidator interface" with too few code and commit comments.  I am having trouble understanding it.

The second line of the commit comment says "Code path that is used with "bFast" flag in CheckForUpdates was extracted into InitializePlugins function" and that explains this function.

It might help to separate a commit that does only this part.  Perhaps "RevalidatePlugins" is also a clearer name for what this does.